### DISCUSSION TITLE
I can't find a html file in project.....

### DISCUSSION THREAD

**Discussion by SuryanshuKushwaha** [Note-Vault/Note-Vault#205] [Q&A] [answered]
.

**Comment by VaibhavArora314** [comment, marked as answer]
That's because we are not using plain html in our project. In this project we are using ejs.

**Comment by SuryanshuKushwaha** [comment]
So how can I see output of code of project on my computer ?

**Reply by VaibhavArora314** [reply]
Make sure you have node installed on your computer. Fork and clone the repository. Then in the repository, run node server.js command. Also create a .env file similar to .env.example in the repository.
